mappings/net/minecraft/util/hit/BlockHitResult.mapping

```diff
@@ -15,13 +16,22 @@ CLASS net/minecraft/class_3965 net/minecraft/util/hit/BlockHitResult
 		ARG 4 blockPos
 		ARG 5 insideBlock
 	METHOD method_17777 getBlockPos ()Lnet/minecraft/class_2338;
+		COMMENT {@return the position of the block that was hit by the raycast}
 	METHOD method_17778 createMissed (Lnet/minecraft/class_243;Lnet/minecraft/class_2350;Lnet/minecraft/class_2338;)Lnet/minecraft/class_3965;
 		ARG 0 pos
 		ARG 1 side
 		ARG 2 blockPos
 	METHOD method_17779 withSide (Lnet/minecraft/class_2350;)Lnet/minecraft/class_3965;
+		COMMENT {@return the modified hit result}
+		COMMENT
+		COMMENT <p>Moves the hit result to a different side of the block.
```

Comment:
The documentation here seems to indicate that the instance is modified and returned. However, the method naming implies that a new instance is created instead. Is the documentation correct here?